### PR TITLE
[UI][fix] - logout page on OCP4

### PR DESCRIPTION
**Pull request by mkralik3**
//test: `@logout-test`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 1]
```diff
@@ -136,7 +136,8 @@ public void logout() {
         $(Element.LOGIN_BUTTON).shouldBe(visible).click();
 
         try {
-            OpenShiftWaitUtils.waitFor(() -> WebDriverRunner.getWebDriver().getCurrentUrl().contains("login"), 20 * 1000);
+            OpenShiftWaitUtils.waitFor(() -> WebDriverRunner.getWebDriver().getCurrentUrl().contains("login") |
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: I wonder if you really want `|` instead of `||`

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Login logout | Log out 
Login logout | Log out and log in 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Login logout | Log out 
Login logout | Log out and log in 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**